src/faldbt/lib.py

```diff
@@ -499,6 +502,18 @@ def null_dump(sql, *multiparams, **params):
     return sqlalchemy.create_mock_engine(url_string, executor=null_dump)
 
 
+def _create_engine_from_connection(adapter: SQLAdapter):
+    if adapter.type() == "postgres":
+        url_string = "postgresql+psycopg2://"
+    else:
+        # TODO: add special cases as needed
+        logger.warn("No explicit url string for adapter {}", adapter.type())
+        url_string = f"{adapter.type()}://"
```

Comment:
Can we use `adapter.credentials.connection_string` here (since it would also contain the prefix)?

Reply:
`adapter.credentials.connection_string` is this a thing?!

Reply:
I recall seeing it somewhere, but now looking deeply I can't find it either. Probably misremembered :/